test/run_test.py

```diff
@@ -893,7 +893,7 @@ def get_selected_tests(options):
     # Only run fx2trt test with specified option argument
     if options.fx2trt_tests:
         selected_tests = list(
-            filter(lambda test_name: "fx2trt" in test_name, selected_tests)
+            filter(lambda test_name: test_name in FX2TRT_TESTS, selected_tests)
         )
     else:
         options.exclude.extend(FX2TRT_TESTS)
```

Comment:
This should be done in a separate PR, but I think it's more clear to not automatically exclude these tests but use the --exclude-fx2trt-tests flag to trigger whether or not to run them.

Reply:
IMO it would be better to skip the tests if PyTorch is not built with TensorRT